app/imports/containers/applyContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import { composeWithTracker } from 'react-komposer';
+
+import ApplyLayout from '/imports/components/apply/applyLayout.jsx';
+
+const composer = ( props, onData ) => {
+  let isLogged = !!Meteor.user() ? true : false;
+  let user = !!Meteor.user ? Meteor.user() : null;
+  let apply = { 
+    section: props.section,
+  };
+  onData( null, { isLogged, user, apply } );
+};
+
+export default composeWithTracker(composer)(ApplyLayout);
```

Comment:
did u read the nice text i wrote for u? it talks about this, this is where the magic happens.
check out the links